packages/mui-base/src/Menu/Root/MenuRoot.tsx

```diff
@@ -139,8 +139,6 @@ namespace MenuRoot {
     delay?: number;
     /**
      * Whether the menu popup opens when the trigger is hovered after the provided `delay`.
-     *
-     * @default nested
```

Comment:
I used to have such a prop in one of the previous implementation attempts and forgot to remove the `@default` annotation. We can add in plain text that by default `openOnHover` is set on nested menus.

Reply:
Updated https://deploy-preview-549--base-ui.netlify.app/base-ui/react-menu/components-api/#menu-root-props